app/main.py

```diff
@@ -31,16 +33,38 @@ def knowledge_panel(
         panels = []
         if facet_tag in HungerGameFilter.list():
             panels.append(
-                hunger_game_kp(hunger_game_filter=facet_tag, value=value_tag, country=country)
+                hunger_game_kp(
+                    hunger_game_filter=facet_tag,
+                    value=value_tag,
+                    sec_facet=sec_facet_tag,
+                    sec_value=sec_value_tag,
+                    country=country,
+                )
             )
         try:
-            panels.append(data_quality_kp(facet=facet_tag, value=value_tag, country=country))
+            panels.append(
+                data_quality_kp(
+                    facet=facet_tag,
+                    value=value_tag,
+                    sec_facet=sec_facet_tag,
+                    sec_value=sec_value_tag,
+                    country=country,
+                )
+            )
         except Exception:
             logging.exception("error occued while appending data-quality-kp")
         try:
-            panels.append(last_edits_kp(facet=facet_tag, value=value_tag, country=country))
+            panels.append(
+                last_edits_kp(
+                    facet=facet_tag,
+                    value=value_tag,
+                    sec_facet=sec_facet_tag,
+                    sec_value=sec_value_tag,
+                    country=country,
+                )
+            )
         except Exception:
-            logging.exception("error occued while appending last-edits-kp")
+            logging.exception("error occued while appending data-quality-kp")
         try:
             if facet_tag in Taxonomies.list():
                 panels.append(wikidata_kp(facet=facet_tag, value=value_tag))
```

Comment:
maybe here we would just like to generate two panels, one for each facet. We may need to specify the "id" we want to avoid having two panels with the same name ?